### PR TITLE
Themeupdate

### PR DESCRIPTION
**Pull request by dragonfaerie**
I did a bunch of theme updates, wound up making the link to fake being a button.  I'll still need to do another round once we decide on a font package.

### CODE REVIEW

**Comment by dragonfaerie** [comment]
@tomizechsterson OK, I fixed 5 conflicted files (I hope, it seems right) and then the tests ran, but failed.  I know why one of them failed, I renamed Destroy to Delete (more user friendly word) but I don't quite get whats going on with the other failure.  And I don't know how to fix it (other than renaming Delete back to Destroy).

File: app/views/characters/index.html.erb [hunk 1]
```diff
@@ -14,9 +14,9 @@
           </p>
         <% end %>
         <p class="card-info">more information in this tag<br>and then theres more<br>perhaps a bit more</p>
-        <%= link_to 'Show', character, :class => "button" %>
-        <%= link_to 'Edit', edit_character_path(character), :class => "button" %>
-        <%= link_to 'Destroy', character, method: :delete, data: { confirm: 'Are you sure?' }, :class => "button" %>
+        <%= link_to '<button type="button" class="button">Show</button>'.html_safe, character, :method => "get" %>
+        <%= link_to '<button type="button" class="button">Edit</button>'.html_safe, edit_character_path(character), :method => "get" %>
+        <%= link_to '<button type="button" class="button">Delete</button>'.html_safe, character, method: :delete, data: { confirm: 'Are you sure?' } %>
```
Comment (tomizechsterson) [inline app/views/characters/index.html.erb:19]: :speech_balloon: I've never seen this before. Is this to get Rails to generate button elements as opposed to `a` elements?

**Review by tomizechsterson — COMMENTED** [review]
Just a question I was wondering about

File: app/views/characters/index.html.erb (hunk 1 above)
Comment (dragonfaerie) [reply app/views/characters/index.html.erb:19]: Yes, that's exactly what that is doing.  It helps keep the buttons rendering the same.  And I probably should have put it on the other link_tos (sigh)

Comment (tomizechsterson) [reply app/views/characters/index.html.erb:19]: Ahh, interesting. Is there any way to style `a`'s like buttons, so we don't have to pass straight-up HTML to `link_to`?

**Review by tomizechsterson — APPROVED** [review]
Sweet!

**Pull request merged**